qemu/tests/cfg/live_snapshot_negative.cfg

```diff
@@ -0,0 +1,24 @@
+- live_snapshot_negative:
+    virt_test_type = qemu
+    type = live_snapshot_negative
+    no raw vmdk qed
+    kill_vm = yes
+    monitor_type = qmp
+    monitors = qmp1
+    source_image = image1
+    snapshot_file = live_snapshot_img
+    snapshot_mode = "absolute-paths"
+    create_cmd = "dd if=/dev/urandom of=%s bs=1M count=1024"
+    file_create = /var/tmp/file
+    clean_cmd = rm -f
+    variants:
+        - simple_test:
```

Comment:
@CongLi  better to drop variant "simple_test",  I think we have no complex negative test in feature.